tests/unit/info.tcl

```diff
@@ -392,6 +392,7 @@ start_server {tags {"info" "external:skip" "debug_defrag:skip"}} {
             set org_qbuf_limit [lindex [r config get client-query-buffer-limit] 1]
             r config set client-query-buffer-limit 1048576
             catch {r set key [string repeat a 1048576]}
+            catch {r read} ; # We sometimes get a broken pipe and I/O error, so make sure there are no more errors
```

Comment:
Nit. The PR top comment is much more comprehensive than this comment. Can you explain both failure modes in a code comments? (write succeeds and read fails, or write fails and the next read fails too as a second error)

Reply:
Sure